### PR TITLE
Mysql2 client support

### PR DESCRIPTION
**Pull request by pacostas**
This PR supports service binding for [mysql2](https://www.npmjs.com/package/mysql2) client.
 
Has also been added:
* Installation/configuration of prettier for keeping a consistency in formating
* Removing [eslint-plugin-standard](https://github.com/nodeshift/kube-service-bindings/commit/06ea804c8de4e2fa2aa93fc599d046d40d9e62bd) due to [deprecation](https://www.npmjs.com/package/eslint-plugin-standard) and not being used anymore from any package. 
* Upgrading [mocha](https://github.com/nodeshift/kube-service-bindings/commit/e2b2039bc68da05f86c6726424739c82520ff4f1) from 9.2.1 to 9.2.2 in order to avoid [snyk PR](https://github.com/nodeshift/kube-service-bindings/pull/32) with outdated package-lock.json
* Filtering values
* Utils folder
* Refactor on setKey function

### CODE REVIEW

**Comment by mhdawson** [comment]
Talked with Costas and we want to update so we can easily remove non-mapped values so that we can avoid warnings from the client when the returned object is passed in to create a connection or pool.

File: clients/MYSQL/mysql2.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  host: 'host',
+  port: 'port',
+  username: 'user',
+  database: 'database',
+  password: 'password'
+};
+
+module.exports = {
+  mapping,
+  filter: (binding) => filterObject(binding, Object.values(mapping))
```
Comment (mhdawson) [inline clients/MYSQL/mysql2.js:13]: I don't think this woud do the right filtering in the general case (where the mapping is not a strict one-to-one), but is ok in this specific case.

**Review by mhdawson — APPROVED** [review]
LGTM

File: clients/MYSQL/mysql2.js (hunk 1 above)
Comment (pacostas) [reply clients/MYSQL/mysql2.js:13]: Yes, so the filter function acts as a wrapper where it allows us to set a custom filtering per client

**Pull request merged**